src/Library/demos/Spin Button/code.rs

```diff
@@ -0,0 +1,41 @@
+use crate::workbench;
+use glib::clone;
+use gtk::glib;
+use gtk::prelude::*;
+use gtk::SpinType::StepForward;
+
+pub fn main() {
+    let hours: gtk::SpinButton = workbench::builder().object("hours").unwrap();
+    let minutes: gtk::SpinButton = workbench::builder().object("minutes").unwrap();
+
+    fn tellTime(hours: &str, minutes: &str) {
```

Comment:
- Please keep this function in the global namespace, below of `main`.
- Also please rename it to `tell_time`